File: Leetcode problems/Leetcode 1528.py
class Solution:
    def restoreString(self, s: str, indices: List[int]) -> str:
        suf=[""]*len(s)
        c=0
        for i in indices:
            suf[i]=s[c]
            c=c+1
        s2=""
        for j in suf:
            s2=s2+j
        return s2

File: Leetcode problems/Leetcode 1528.js
/**
 * @param {string} s
 * @param {number[]} indices
 * @return {string}
 */
var restoreString = function(s, indices) {
    s2=new Array(s.length);
    count=0;
    while(count<s.length){
        pos=indices.at(count);
        s2[pos]=s[count];
        count=count+1;
    }
    return s2.join('');
};
